README.md

```diff
@@ -31,7 +31,15 @@ Please review [system requirements](/docs/system-requirements.md) before proceed
 <a name="updates"></a>
 ## Updates
 
-Switch to your `<projects>` folder and run:
+Switch to your `<projects>` folder.
+
+If Drude is running, run:
+
+```
+dsh halt
```

Comment:
We had a fix for this committed recently, so no need in these lines
https://github.com/blinkreaction/drude/commit/f9a88cf7fd5f95361efa80d0acc6bcfae4ecac9f